README.md

```diff
@@ -414,52 +429,23 @@ A subscription allows you to continue paginating a collection until new items ar
 
 A subscription fetches pages until the first empty page is reached. From that point, it fetches a new page at an interval in milliseconds defined by config (`{ interval: 5000 }`).  This interval defaults to 5000 milliseconds.  A subscription object is returned.  To terminate polling, call `unsubscribe()` on the subscription object.
 
-#### Simple subscription
-
-```javascript
-const subscription = collection.subscribe({
-  next(item) {
-    console.log('Process item');
-  },
-  error(err) {
-    console.log('Handle error');
-    console.log('Call subscription.unsubscribe() to terminate');
-  }
-});
-```
-
-#### Advanced subscription
+Depending on the polling interview you choose, you may run into rate limiting exceptions.  In that case you should enable our rate limiting retry strategy, see [Rate Limiting](#rate-limiting).
 
-Rate-limiting errors are common with subscriptions. Here's one way to handle them:
+#### Simple subscription example
 
 ```javascript
 const subscription = collection.subscribe({
-  interval: 5000, // Time in ms before fetching new logs when all existing logs are read
+  interval: 5000,
   next(item) {
-    console.log('Process item');
+    console.log(item);
   },
   error(err) {
-    if (err.status == 429) {
-      const retryEpochMs = parseInt(err.headers.get('x-rate-limit-reset'), 10) * 1000;
-      const retryDate = new Date(retryEpochMs);
-      const nowDate = new Date(err.headers.get('date'));
-      const delayMs = retryDate.getTime() - nowDate.getTime();
-
-      console.log('backoff', retryDate, now, delayMs)
-      return new Promise(resolve => {
-        setTimeout(resolve, delayMs);
-      });
-    }
-
-    if (err.status >= 500) {
-      subscription.unsubscribe();
-    }
-  },
-  complete() {
-    console.log('subscription.unsubscribe() was called');
-    console.log('next() will no longer be triggered');
+    // handle error
   }
 });
+
+// In the future, unsubscribe as needed:
+subscription.unsubscribe()
```

Comment:
Since there are a couple of instances where `unsubscribe` can be called, it may be more clear to pull this out into an **unsubscribe from a subscription** header.

That way, we can give more thorough examples by calling `unsubscribe` when:
1. The `error` handler is involked

    ```javascript
    const subscription = collection.subscribe({
      interval: 5000,
      next(item) {
        console.log(item);
      },
      error(err) {
        // handle error and unsubscribe if necessary
        subsciption.unsubscribe();
      }
    });
    ``` 
2. Unsubscribe in the [`completion`](https://github.com/okta/okta-sdk-nodejs/#system-log) handler.

Reply:
RE 2, you don't call unsubscribe in the completion handler, rather that handler gets called if you call unsubscribe.  So I'm not sure that we really have more examples to document?